src/steamship/base/client.py

```diff
@@ -141,7 +141,7 @@ def dict(self, **kwargs) -> dict:
         # Because of the trick we do to hack these in as both static and member methods (with different
         # implementations), Pydantic will try to include them by default. So we have to suppress that otherwise
         # downstream serialization into JSON will fail.
-        if "exclude" not in kwargs:
+        if "exclude" not in kwargs:  # TODO (enias): Review
```

Comment:
I think I've found an alternative solution. The new solution is part of this PR for testing, will circle back with a new PR when I confirmed it works as expected.

Reply:
👍